app/models/robots/pooling_and_splitting_robot.rb

```diff
@@ -0,0 +1,120 @@
+# frozen_string_literal: true
+
+module Robots
+  class PoolingAndSplittingRobot < Robot # rubocop:todo Style/Documentation
```

Comment:
Don't leave the documentation todo hanging around, write a little bit about what the robot does, and how it differs from the others.

Reply:
Done